doc/en/components/layouts/dock-manager.md

```diff
@@ -330,6 +330,7 @@ layout.rootPane.orientation = IgcSplitPaneOrientation.vertical;
 this.dockManager.layout = { ...layout };
 ```
 
+<!-- WebComponents -->
 ### Adding Panes At Runtime
```

Comment:
how about build flagging rest of subsection in Usage? They are missing samples/code snippets